core/server/worker/src/main/java/alluxio/worker/grpc/DoraWorkerClientServiceHandler.java

```diff
@@ -51,6 +51,8 @@ public class DoraWorkerClientServiceHandler extends BlockWorkerGrpc.BlockWorkerI
 
   private static final boolean ZERO_COPY_ENABLED =
       Configuration.getBoolean(PropertyKey.WORKER_NETWORK_ZEROCOPY_ENABLED);
+  private static final int LIST_STATUS_BATCH_SIZE =
+      Configuration.getInt(PropertyKey.MASTER_FILE_SYSTEM_LISTSTATUS_RESULTS_PER_MESSAGE);
```

Comment:
I think we need a dedicated property key for Dora. This way users won't get confused about the configurations.

Reply:
I was thinking about this, too.

As I replied in https://github.com/Alluxio/alluxio/pull/17058, I am reluctant to add more config keys.
Maybe I can change MASTER_FILE_SYSTEM_LISTSTATUS_RESULTS_PER_MESSAGE to 
USER_FILE_SYSTEM_LISTSTATUS_RESULTS_PER_MESSAGE?
So it can be shared by Master and Worker.
Because this configure key is required by Alluxio Client.
Please comment.

Reply:
`MASTER_FILE_SYSTEM_LISTSTATUS_RESULTS_PER_MESSAGE` has `MASTER` scope. This configuration won't be picked up by worker when it's loading cluster default configuration.

See
https://github.com/Alluxio/alluxio/blob/35820f8d61283b3200f6019917384e367698a585/core/common/src/main/java/alluxio/conf/PropertyKey.java#L3812-L3818

https://github.com/Alluxio/alluxio/blob/50a282e5f48aa00b6d3e83dd1b3d60079162dcea/core/server/worker/src/main/java/alluxio/worker/AlluxioWorker.java#L61

Either we change its scope to `SERVER`, or we can define a new config key. Such key should be hidden to users, as long as the default value makes sense for the most common cases.

Reply:
But I tested to add a config key in conf/alluxio-site.properties:
```
HuanghuaMacBookPro:alluxio huanghua$ tail -n 1 conf/alluxio-site.properties
alluxio.master.filesystem.liststatus.result.message.length=17
HuanghuaMacBookPro:alluxio huanghua$ 

```

And this value can be loaded here correctly by Worker process.
Why?